lib/Controller/TableScheme.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
```

Comment:
Just rebased but this one should fail due to missing copyright headers. Best would be to always rebase to latest main before doing another push on a pr, so all checks and changes are up to date with the base branch :)